lib/embedded_tweet_view.dart

```diff
@@ -207,6 +213,25 @@ class EmbeddedTweetView extends StatelessWidget {
                             color: (darkMode)
                                 ? Colors.grey[400]
                                 : Colors.grey[600]))),
+                if (showRepliesCount && _tweetVM.repliesCount != null)
```

Comment:
I think this can be extracted to some getter and reused below

Reply:
What do you think about the following?

I'm thinking of adding the following method on the VM.

`bool shouldShowReplies(bool shouldShow) => shouldShow && repliesCount != null; //I'll come up with a better param name if we are to ahead with this :)`

and then on the tweet view we'll do:

`if (_tweetVM.shouldShowReplies(showRepliesCount)){
       //show replies
}`

Reply:
I was more thinking about a getter in EmbeddedTweetView

```bool get _shouldShowReplies => showRepliesCount && _tweetVM.repliesCount != null```

Reply:
Ok, that's fine, I thought maybe what I proposed above will be equally reusable should the need arise to add this feature to other views as well. Compact and Tweet.